src/main/java/org/broadinstitute/hellbender/utils/LoggingUtils.java

```diff
@@ -87,6 +93,22 @@ private static void setJavaUtilLoggingLevel(final Log.LogLevel verbosity) {
         return javaUtilLevelNamespaceMap.get(picardLevel);
     }
 
+    /**
+     * Send log output to file if file name is not null.
+     *
+     * @param fileName  File path for log output
+     */
+    public static void sendLogToFile(final String fileName) {
```

Comment:
1.  I would make this expect a non-null filename and have the null-check be at the call site I think.
2. If you call this repeatedly with multiple files can you direct the log to multiple files?  
3. Does this suppress log messages on the console?